components/grid/compact-grid.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Grid Size - Compact Grid
+page_title: Grid Size | Grid for Blazor
+description: Adjust the size and modify the appearance of the Telerik UI for Blazor Grid.
+slug: compact-grid
+tags: telerik,blazor,grid,size,compact
+published: true
+position: 20
```

Comment:
I suggest changing the position to place this article after the key component features such as paging, sorting, filtering, grouping. We may add it after the Toolbar, for example.